### PR TITLE
Comment out supplied name column [#151787259]

### PR DESCRIPTION
**Pull request by raivop**

### CODE REVIEW

File: grails-app/views/speciesListItem/_list-tab.gsp [hunk 1]
```diff
@@ -8,11 +8,13 @@
                     titleKey="general.scientificName"
                 />
 
+                <!--
```
Comment (alarku07) [inline grails-app/views/speciesListItem/_list-tab.gsp]: why not use comment blocks that are not rendered?

**Comment by alarku07** [comment]
just asking - did you check that it doesn't break gallery modal's footer info?

File: grails-app/views/speciesListItem/_list-tab.gsp (hunk 1 above)
Comment (raivop) [reply grails-app/views/speciesListItem/_list-tab.gsp]: To allow spontaneous fiddling but now I think about it: that's not going to work.

**Comment by raivop** [comment]
No. Why would it?

**Comment by alarku07** [comment]
the footer data depends on what columns are shown in list and since it's ala, any unforseen magic can happen

**Pull request merged**